### ISSUE TITLE
[Nightly] Placing of menu entries for "individual route" in conditional map symbol is not obvious

### ISSUE THREAD

**Issue by MagpieFourtyTwo** [cgeo/cgeo#11896]
### Describe your problem!

Created an individual route (IR), used c:geo some time - and after some actions the menu entries for sorting and esp. clearing the IR got lost. Now I have an IR displayed in the map, it's distance is displayed as well, but have no possibility to get rid of it (other than tapping all points of it, one after the other). Menu says "Load individual route" only.

Strange enough: If I select the load route menu entry, the following dialog says "Overwrite existing route?"

### How to reproduce?

.

### Reproducible

Unclear

### c:geo Version

2021.10.15-NB-fd3f98e

**Comment by MagpieFourtyTwo** [comment]
> A vanishing menu entry "magically reappearing" in the popup menu might confuse users

Maybe - but how come that a vanishing map symbol is less confusing than a vanishing menu entry ...? ;)
Esp. as the always existing burger menu entry "Load IR" seems to suggest that other IR functions may be accessible there, too ...?

**Comment by moving-bits** [comment]
@MagpieFourtyTwo 

IMHO this is superseded by the recent changes regarding tracks/routes and individual routes, see #12129 (implemented with #12499, #12507, with follow-ups in #12584 and #12586): all track/route and individual route related actions are in a bottom sheet dialog (similar to map quick menu), accessible both via a quick access bottom at the map bottom and via a menu entry "Routes / Tracks". Ok to close?

![image](https://user-images.githubusercontent.com/3754370/150653346-394bed08-fb0c-4b45-917b-cf1cfcfc3e70.png)

**Comment by MagpieFourtyTwo** [comment]
I would be ok with closing - although the current implementation seems to be a bit inconsistent, IMHO, as the route "style" (show/hide, direct, walk, bike, car), which is *the* main routing action (or at least the most changed one), is still in the quick settings menu ... so perhaps the routing styles should move to the routing menu, too, and the bottom route menu symbol should appear for "normal" cache routes, too ...? Even more as these action buttons determine the individual route style, too ...? Although the "show/hide" does only hide "normal" cache routes ...?

BTW: The "Hide Individual route" button seems to do nothing at all, at least over here ... strange.

**Comment by moving-bits** [comment]
> BTW: The "Hide Individual route" button seems to do nothing at all, at least over here ... strange.

Hm, that has worked already. Will look into it. While working on a different issues I've just stumbled upon a freshly loaded individual route only being listed in the bottom sheet dialog after closing and reloading, so in both cases something seems to be wrong with updating. I'll open a separate issue for it and look into it.

Placing the routing mode buttons is a bit difficult, as it does apply to both individual routes and targets set otherwise, so that we decided to leave it in the regular quick settings menu for the time being.
